anitya/lib/model.py

```diff
@@ -602,3 +602,129 @@ class ProjectVersion(BASE):
     version = sa.Column(sa.String(50), primary_key=True)
 
     project = sa.orm.relation('Project', backref='versions_obj')
+
+
+class ProjectFlag(BASE):
+    __tablename__ = 'projects_flags'
+
+    id = sa.Column(sa.Integer, primary_key=True)
+
+    project_id = sa.Column(
+        sa.Integer,
+        sa.ForeignKey(
+            "projects.id",
+            ondelete="cascade",
+            onupdate="cascade")
+    )
+
+    project = sa.orm.relation('Project')
```

Comment:
I normally put the relation at the end of the table definition, first all the column, then the constraints, then the relations